proxygen/lib/http/codec/compress/experimental/simulator/CompressionScheme.h

```diff
@@ -52,12 +55,24 @@ class CompressionScheme : public folly::EventBase::LoopCallback {
   void runLoopCallback() noexcept override;
 
   /* List of blocks encoded in the current event loop */
-  std::list<std::tuple<bool, std::unique_ptr<folly::IOBuf>,
+std::list<std::tuple<FrameFlags, bool /*newPacket*/, std::unique_ptr<folly::IOBuf>,
```

Comment:
I'm going to put some style nits in here.  Only bother fixing them if you are making another rev for something more substantial.  Otherwise I can fix it when I merge.

nit - we have an 80 column limit.  Consider using BlockInfo = std::tuple<....>